### PR TITLE
manifests: include Oneplus5/5T (cheeseburger & dumpling)

### PR DESCRIPTION
**Pull request by Vince1171**
should I use an ssh remote instead of https? or a git?
```<remote name="Flohack74" fetch="ssh://git@github.com/Flohack74" revision="halium-9.0" />```
```<remote name="Flohack74" fetch="git://github.com/Flohack74" revision="halium-9.0" />```

### CODE REVIEW

**Comment by Flohack74** [comment]
You do not need a remote at all, just write <project path="device/oneplus/msm8998-common" name="Flohack74/android_device_oneplus_msm8998-common" > and it should work. Github and halium-9.0 branch name are default for this manifest, and so you only need to mention the username in front of the repo name ;)

File: manifests/oneplus_cheeseburger.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+
+  <!-- Linux kernel 4.4 (Android) -->
+  <project path="kernel/oneplus/msm8998" name="Flohack74/android_kernel_oneplus_msm8998" />
+
+  <!-- Device configuration -->
+  <project path="device/oneplus/msm8998-common" name="Flohack74/android_device_oneplus_msm8998-common" />
+  <project path="device/oneplus/cheeseburger"   name="Flohack74/android_device_oneplus_cheeseburger" />
```
Comment (Flohack74) [inline manifests/oneplus_cheeseburger.xml]: Remove excess spaces.. also below

Comment (Vince1171) [reply manifests/oneplus_cheeseburger.xml]: Done

**Comment by Flohack74** [comment]
I was wrong, sorry. revision="halium-9.0" is needed, it defaults to lineage-16.0 otherwise ^^

**Comment by fredldotme** [comment]
LGTM

**Review by fredldotme — APPROVED** [review]

**Pull request merged**